ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/ComposeFormFields/ComposeFormFields.tsx

```diff
@@ -77,11 +93,17 @@ export const ComposeFormFields = ({
           type="number"
           wrapperClassName="u-sv2"
         />
-        <FormikField
-          label="Enable hugepages"
-          name="hugepagesBacked"
-          type="checkbox"
-        />
+        <Tooltip
+          data-test="hugepages-tooltip"
+          message={getHugepagesTooltip(isLxd, hasFreeHugepages)}
```

Comment:
Does an empty tooltip should a blank black box?

Reply:
Nah if the message is falsy it doesn't render anything